packages/redis/src/index.ts

```diff
@@ -21,6 +21,24 @@ export interface IRedisOptions {
     readonly db: number;
     readonly encoder: IMessageEncoder;
     readonly typeMapper: IMessageTypeMapper;
+    readonly writeStream?: string;
```

Comment:
not sure if you wanna add these here as optional ... then the exisiting things that only expect `IRedisOptions` will suddenly have these optional fields available that don't do anything ... your union type `IRedisInputStreamOptions` should work without having them here as optional though